src/EditorFeatures/TestUtilities/Traits.cs

```diff
@@ -38,7 +38,8 @@ public static class Features
             public const string CodeActionsChangeToIEnumerable = "CodeActions.ChangeToIEnumerable";
             public const string CodeActionsChangeToYield = "CodeActions.ChangeToYield";
             public const string CodeActionsConvertToInterpolatedString = "CodeActions.ConvertToInterpolatedString";
-            public const string CodeActionsConvertToIterator = "CodeActions.CodeActionsConvertToIterator";
+            public const string CodeActionsConvertToIterator = "CodeActions.ConvertToIterator";
+            public const string CodeActionsConvertToSwitch = "CodeActions.ConvertToSwitch";
```

Comment:
Actually I did it and revert it! following others like ConvertToInterpolation, ConvertToIterator, etc.